gef-ui/templates/application-abandon.njk

```diff
@@ -0,0 +1,61 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+
+{% block pageTitle %}
+  Abandon application
+{% endblock %}
+
+{% block content %}
+
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row" data-cy="application-submit-page">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-6" data-cy="main-heading">
+        Confirm that you want to abandon
+      </h1>
+
+      <p class="govuk-body govuk-!-margin-bottom-6">
+        You’ll still be able to find this {% if application.isAutomaticCover %} notice {% else %} application {% endif %} in the abandoned folder. 
+      </p>
+    </div>
+
+    <div class="govuk-grid-column-one-half-from-desktop">
+      <form
+        method="post"
+        data-cy="form">
+
+        <p class="govuk-button-group govuk-!-margin-top-7" style="vertical-align: middle">
```

Comment:
is the inline style needed?

Reply:
good catch ..  I think that was leftover from when it was a link. button-group resolves this so will remove